CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.8.0 (2021-04-14)
+
+- Updated the code examples to demonstrate adding new items to the bottom of a repeater list. [#228](https://github.com/blackbaud/skyux-lists/pull/228)
```

Comment:
You don't need a release for updating the docs - those get released when merged. If you go to the live docs, you should be able to already see your changes.